Added Carthage to the readme

RuiAAPeres opened the pull request:
Since the target from where the framework is built  is already a shared, you get carthage for free

RuiAAPeres committed:
Added Carthage to the readme

Since the target from where the framework is built  is already a shared, you get carthage for free

devxoul commented:
Carthage is only available for iOS8+ targets.

> Note that Carthage only supports dynamic frameworks, which are only available on iOS 8 or later (or any version of OS X).

How about [this](https://github.com/devxoul/JLToast#installation) approach?

RuiAAPeres committed:
Fixed the formatting

Also pushed Carthage above CocoaSeeds, so the approaches for iOS8+ are together. Removed the "with Podfile" "with Cartfile" "with SeedFile", since people who use any of these approaches would know the corresponding file name.

RuiAAPeres committed:
Added the version

So it's consistent with the other 2 examples

devxoul commented:
@RuiAAPeres, Thank you!
